Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Daha büyük başarım ekranı\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.animatica": "Optifine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bobby": "Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
+  "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
+  "modmenu.summaryTranslation.capes": "Optifine veya diğerlerinden pelerinler\n§9✎§8 Türler, tercih edilen tür, elitra, ...",
+  "modmenu.summaryTranslation.citresewn": "Optifine'ın özel eşyaları\n§9✎§8 Modu aç/kapa, bozuk paketlere izin ver, ...",
+  "modmenu.summaryTranslation.colormatic": "Optifine'ın özel renkleri\n§9✎§8 Gökyüzünü veya yer boşluğunu temizle, ışık yumuşatma",
+  "modmenu.summaryTranslation.continuity": "Optifine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.cullleaves": "Optifine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
+  "modmenu.summaryTranslation.cem": "Optifine'ın özel varlıkları (alfa)\n§9✎§8 Optifine desteği, model düzeltimi",
+  "modmenu.summaryTranslation.dcch": "Farklı dünyalardan yazım geçmişini tut\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.dynamicfps": "Arkaplandayken FPS'i düşür\n§9✎§8 Modu aç/kapa, üzerindeyken normale çevir, ...",
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandıklar, tabelalar, yataklar vb.\n§9✎§8 Geliştirilecek bloklar",
+  "modmenu.summaryTranslation.entityculling": "Engellerin arkasındaki mobları görüntüleme\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.entity_texture_features": "Optifine'daki rastgele ve ışık emen moblar\n§9✎§8 Rastgele moblar, ışık emiciliği, göz kırpabilmeleri, ...",
+  "modmenu.summaryTranslation.fabrishot": "F9 ile yüksek çözünürlüklü görüntü al\n§9✎§8 F2 tuşunu, çözünürlüğü, tuş ataması vb. üzerine yaz.",
+  "modmenu.summaryTranslation.farsight": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
+  "modmenu.summaryTranslation.fastopenlinksandfolders": "Bağlantı veya klasörleri açarken donmayı önler\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.forgetmechunk": "DFixes lag spikes on chunk borders\n§8✎ (seçenekleri yok)",
```

Comment:
Seems like you forgot to translate

Reply:
Yeah, ironically and coincidentally, I forgot to the translate the description of *Forgot Me* Chunk mod, haha. Thanks for notice!